core/src/com/unciv/ui/mapeditor/MapEditorLoadTab.kt

```diff
@@ -118,7 +118,17 @@ class MapEditorLoadTab(
                         map.mapParameters.mods -= modBaseRuleset
                     }
 
-                    editorScreen.loadMap(map)
+                    val ruleset = RulesetCache.getComplexRuleset(map.mapParameters)
+                    val rulesetIncompatibilities = map.getRulesetIncompatibility(ruleset)
+                    if (rulesetIncompatibilities.isNotEmpty()) {
+                        map.removeMissingTerrainModReferences(ruleset)
+                        val message = "{This map has errors:}\n\n".tr() +
+                                rulesetIncompatibilities.sorted().joinToString("\n") { it.tr() } +
```

Comment:
So you first sort and then translate? Would that still look good in non-English languages?

Reply:
No oversight, I was reluctant to pull the collator for something as rarely seen. I wanted it deterministic and a bit grouped, actual alphabetics - shrug.